rxbinding/src/androidTest/java/com/jakewharton/rxbinding/view/RxViewGroupTestActivity.java

```diff
@@ -0,0 +1,19 @@
+package com.jakewharton.rxbinding.view;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+
+public final class RxViewGroupTestActivity extends Activity {
+  ViewGroup viewGroup;
```

Comment:
Just make this type `LinearLayout` so you don't have to cast in the test `before()` method